src/abstractdataframe/show.jl

```diff
@@ -13,6 +13,17 @@ function ourstrwidth(io::IO, x::Any, buffer::IOBuffer)
     return textwidth(String(take!(buffer)))
 end
 
+function trimstring32(s::AbstractString)
```

Comment:
`trim` in the context of strings usually means dropping whitespace at the beginning and end AFAIK. Maybe `truncate`?

Reply:
OK - I will change to `truncate`.